### ISSUE TITLE
Bug : Cannot open any tab 4.6.4

### ISSUE THREAD

**Issue by georgeplaton7** [sirensolutions/kibi#73]
### Versions

Kibi: 4.6.4 Community
Kibana: 4.6.1

Operating System: Linux x64

### Behavior 

I can see all the Logstash, MetricBeat indexes and data on Kibana. We are using XPack to secure our cluster + Kibana.
Kibi can fetch the name of the visualizations, but he cannot pull any data from ElasticSearch/Kibana.
On the console I keep getting
````
Error: IndexPattern's configured pattern does not match any indices
    at handleMissingIndexPattern (kibana.bundle.js?v=13675:39985)

````

#### Expected:

Too see some nice graphs, so I'm motivated to go for the Enterprise version ? :-)

#### Actual:
See Behaviour.

Logs :
````
 log   [02:15:16.131] [info][status][plugin:heatmap@0.1.0] Status changed from uninitialized to green - Ready
  log   [02:15:16.135] [info][status][plugin:kibana@1.0.0] Status changed from uninitialized to green - Ready
  log   [02:15:16.197] [info][status][plugin:elasticsearch@1.0.0] Status changed from uninitialized to yellow - Waiting for Elasticsearch
  log   [02:15:16.211] [info][status][plugin:kibi_radar_vis@4.5.3] Status changed from uninitialized to green - Ready
  log   [02:15:16.215] [info][status][plugin:kibi_timeline_vis@4.6.4] Status changed from uninitialized to green - Ready
  log   [02:15:16.224] [info][status][plugin:kibi_wordcloud_vis@4.6.3-1] Status changed from uninitialized to green - Ready
  log   [02:15:16.227] [info][status][plugin:kbn_vislib_vis_types@1.0.0] Status changed from uninitialized to green - Ready
  log   [02:15:16.233] [info][status][plugin:kibi_core@1.0.0] Status changed from uninitialized to yellow - Initialising the query engine
  log   [02:15:16.254] [info][status][plugin:kibi_data_table_vis@1.0.0] Status changed from uninitialized to green - Ready
  log   [02:15:16.258] [info][status][plugin:kibi_query_viewer_vis@1.0.0] Status changed from uninitialized to green - Ready
  log   [02:15:16.264] [info][status][plugin:kibi_sequential_join_vis@1.0.0] Status changed from uninitialized to green - Ready
  log   [02:15:16.268] [info][status][plugin:markdown_vis@1.0.0] Status changed from uninitialized to green - Ready
  log   [02:15:16.274] [info][status][plugin:metric_vis@1.0.0] Status changed from uninitialized to green - Ready
  log   [02:15:16.289] [info][status][plugin:migrations@1.0.0] Status changed from uninitialized to yellow - Checking for out of date objects.
  log   [02:15:16.330] [info][status][plugin:elasticsearch@1.0.0] Status changed from yellow to green - Kibana index ready
  log   [02:15:16.336] [info][status][plugin:saved_objects_api@1.0.0] Status changed from uninitialized to green - Ready
  log   [02:15:16.340] [info][status][plugin:spyModes@1.0.0] Status changed from uninitialized to green - Ready
  log   [02:15:16.345] [info][status][plugin:statusPage@1.0.0] Status changed from uninitialized to green - Ready
  log   [02:15:16.350] [info][query_engine] Found kibi index
  log   [02:15:16.352] [info][status][plugin:table_vis@1.0.0] Status changed from uninitialized to green - Ready
  log   [02:15:16.361] [info][listening] Server running at https://0.0.0.0:5606
  log   [02:15:16.422] [info][query_engine] Loading templates
  log   [02:15:16.434] [info][kibi_core] { message: 'QueryEngine initialized successfully.' }
  log   [02:15:16.436] [info][status][plugin:kibi_core@1.0.0] Status changed from yellow to green - Query engine initialized
  log   [02:15:16.439] [warning][query_engine] Template [kibi-json-jade] already exists
  log   [02:15:16.439] [warning][query_engine] Template [kibi-table-handlebars] already exists
  log   [02:15:16.439] [warning][query_engine] Template [kibi-table-jade] already exists
  log   [02:15:16.458] [info][status][plugin:migrations@1.0.0] Status changed from yellow to green - All objects are up to date.


````

### Steps to reproduce the problem

Use Kibana 4.6.1 with XPack + Kibi 4.6.4

I am missing something in the configuration ?

**Comment by fabiocorneti** [comment]
Hello,
a few questions:

- Have you installed the Kibana shield plugin in Kibi or Kibi is running without additional plugins?
- Do you see any red error in the UI or just a blank page?
- Can you take a screenshot of Dashboard page and the Settings -> Index patterns page?

It would also be helpful to see if there are network requests with a status different than 200 in the network inspector, e.g. :

![image](https://cloud.githubusercontent.com/assets/32792/24207897/0ae1544e-0f1a-11e7-8d80-cbcb6437db58.png)

I tried to replicate the issue by restoring a .kibi with several index patterns to an empty ES 2.4.4 + Shield instance but I have not been able to trigger the error; it might be caused by missing permission to either the kibi server user or the frontend user, the list of failed requests might provide a hint if that's the case.

**Comment by szydan** [comment]
Hi @qgicup we've just released 4.6.4-4
https://github.com/sirensolutions/kibi/releases/tag/tag-4.6.4-4
Which you might wish to try 
Closing this issue as there was no feedback for more than 3 months 
Feel free to reopen
